packages/loader/container-loader/src/loader.ts

```diff
@@ -257,6 +257,26 @@ export type IDetachedBlobStorage = Pick<IDocumentStorageService, "createBlob" |
 	getBlobIds(): string[];
 };
 
+/**
+ * With an already-resolved container, we can request a component directly, without loading the container again
+ * @param container - a resolved container
+ * @returns component on the container
+ */
+export async function requestResolvedObjectFromContainer(
+	container: IContainer,
+): Promise<IResponse> {
```

Comment:
i think adding an optional headers param could be useful, if they need to pass any.  `headers?: IRequestHeader`

Reply:
some examples of the types of headers someone may want to pass: https://github.com/microsoft/FluidFramework/blob/main/packages/runtime/container-runtime/src/containerRuntime.ts#L1570L1578